frontend/packages/knative-plugin/src/actions/add-event-source.tsx

```diff
@@ -1,13 +1,21 @@
 import * as React from 'react';
-import { HelpIcon } from '@patternfly/react-icons';
 import {
   createKebabAction,
   KebabAction,
 } from '@console/dev-console/src/utils/add-resources-menu-utils';
 import { ImportOptions } from '@console/dev-console/src/components/import/import-types';
+import * as eventSourceImg from '../imgs/event-source.svg';
+
+const eventSourceIconStyle = {
+  width: '1em',
+  height: '1em',
+};
+const EventSourceIcon: React.FC = () => {
+  return <img style={eventSourceIconStyle} src={eventSourceImg} alt="" />;
```

Comment:
remove alt if we don't have value to put.
```suggestion
  return <img style={eventSourceIconStyle} src={eventSourceImg} />;
```

Reply:
Added to keep lint happy, and alt here won't be needed as we do have title next to it